src/Files.App/Actions/Navigation/OpenInNewWindowItemAction.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+using Windows.System;
+
+namespace Files.App.Actions
+{
+	internal class OpenInNewWindowItemAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context;
+
+		public string Label
+			=> "OpenInNewWindow".GetLocalizedResource();
+
+		public string Description
+			=> "OpenInNewWindowItemActionDescription".GetLocalizedResource();
+
+		public RichGlyph Glyph
+			=> new(opacityStyle: "ColorIconOpenInNewWindow");
+
+		public bool IsExecutable
```

Comment:
And this as long as only folders are selected.

Reply:
Fixed